lib/ai/process-chat-response.ts

```
@@ -0,0 +1,371 @@
+// derived from https://github.com/vercel/ai/blob/93bd8992e496edf28424e766b6f31313ee054e5d/packages/ui-utils/src/process-chat-response.ts
+import { LanguageModelV1FinishReason } from '@ai-sdk/provider';
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)